modules/log6x-installing-the-logging-ui-plug-in-cli.adoc

```diff
@@ -0,0 +1,52 @@
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-04-18
+:_mod-docs-content-type: PROCEDURE
+
+[id="installing-the-logging-ui-plug-in-cli_{context}"]
+= Installing the Logging UI plug-in by using the CLI
```

Comment:
Supplementary Style Guide: https://redhat-documentation.github.io/supplementary-style-guide/#plug-in
```suggestion
= Installing the Logging UI plugin by using the CLI
```